client/src/app/[locale]/(general)/contacts/messengers.tsx

```diff
@@ -11,24 +11,24 @@ export const Messenger = ({ title, link, icon: IconSVG }: IMessengerMock) => {
     : `https://www.${link}`;
 
   return (
-    <div className="tablet:w-[48%] desktop:w-auto">
+    <div className="tablet:w-[49%] desktop:w-auto">
```

Comment:
I'm not sure that's a good approach. I suggest we reconsider. Why 49%? Wouldn't it be better to make it 50% - some kind of gap?